nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/trash/test/AbstractTestTrashService.java

```diff
@@ -124,7 +124,7 @@ public void testBase() {
         assertFalse(trashService.canPurgeOrUntrash(Collections.singletonList(doc1), principal));
         assertFalse(trashService.canPurgeOrUntrash(Collections.singletonList(doc2), principal));
 
-        TrashInfo info = trashService.getTrashInfo(Arrays.asList(fold, doc1, doc3), principal, false, false);
+        TrashInfo info = (TrashInfo) trashService.getTrashInfo(Arrays.asList(fold, doc1, doc3), principal, false, false);
```

Comment:
Do you need to cast ? An import of `TrashInfo` from core api should work, but IDE won't propose this as your in the same package than a class matching the name.